*** testsFinal/A-05-01-M-ok.out	2021-11-08 16:51:37.257588529 +0000
--- testsFinal/A-05-01-M-ok.outhyp	2021-11-08 19:36:00.228817845 +0000
***************
*** 34,46 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Visualizar lotes fornecidos por parceiro: Operação inválida: O parceiro 'MM1' não existe.
! Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Abrir
  2 - Guardar
--- 34,46 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu Principal
  1 - Abrir
  2 - Guardar
