*** testsFinal/A-06-03-M-ok.out	Sun Nov  7 16:04:59 2021
--- testsFinal/A-06-03-M-ok.outhyp	Tue Nov  9 13:09:34 2021
***************
*** 15,32 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: Identificador do produto: HIDROGENIO|M1|2|5000
! HIDROGENIO|M1|20|5000
! HIDROGENIO|M1|30|5000
! HIDROGENIO|S1|20|5000
! HIDROGENIO|S1|50|5000
! HIDROGENIO|S1|200|5000
! Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Abrir
  2 - Guardar
--- 15,27 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu Principal
  1 - Abrir
  2 - Guardar
