*** testsFinal/A-10-02-M-ok.out	2021-11-08 16:51:37.261590529 +0000
--- testsFinal/A-10-02-M-ok.outhyp	2021-11-09 18:39:40.718923104 +0000
***************
*** 26,32 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Identificador do produto: Activa/desactivar notificações de um produto: Operação inválida: O parceiro 'MM1' não existe.
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 26,32 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Identificador do produto: Activa/desactivar notificações de um produto: Operação inválida: O parceiro 'SAL' não existe.
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
