*** testsFinal/A-12-03-M-ok.out	2021-11-08 16:51:37.273596530 +0000
--- testsFinal/A-12-03-M-ok.outhyp	2021-11-11 00:17:54.148780846 +0000
***************
*** 120,127 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: -60000
! Saldo contabilístico: -60000
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 120,127 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 0
! Saldo contabilístico: 0
  Menu Principal
  1 - Abrir
  2 - Guardar
