*** testsFinal/A-12-03-M-ok.out	Sun Nov  7 16:04:59 2021
--- testsFinal/A-12-03-M-ok.outhyp	Thu Nov 11 10:30:48 2021
***************
*** 120,127 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: -60000
! Saldo contabilístico: -60000
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 120,127 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 0
! Saldo contabilístico: 0
  Menu Principal
  1 - Abrir
  2 - Guardar
