*** testsFinal/A-13-02-M-ok.out	Sun Nov  7 16:04:59 2021
--- testsFinal/A-13-02-M-ok.outhyp	Thu Nov 11 11:28:13 2021
***************
*** 28,34 ****
  0 - Sair
  Escolha uma opção: HIDROGENIO|2|5000
  ROLHA|2|500
! SAL|1|0
  VIDRO|1|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
--- 28,34 ----
  0 - Sair
  Escolha uma opção: HIDROGENIO|2|5000
  ROLHA|2|500
! SAL|1|10
  VIDRO|1|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
***************
*** 53,58 ****
--- 53,59 ----
  0 - Sair
  Escolha uma opção: Preço: HIDROGENIO|S1|2|5000
  ROLHA|M1|2|500
+ SAL|S1|1|10
  VIDRO|M1|1|500
  Consultas
  1 - Produtos com Preço Abaixo de Limite
