<<<<<<< HEAD
*** testsFinal/A-14-01-M-ok.out	Wed Nov 10 21:22:28 2021
--- testsFinal/A-14-01-M-ok.outhyp	Wed Nov 10 21:53:28 2021
=======
*** testsFinal/A-14-01-M-ok.out	2021-11-08 16:51:37.277598530 +0000
--- testsFinal/A-14-01-M-ok.outhyp	2021-11-10 21:57:16.920969939 +0000
>>>>>>> ee1cb16a9655cf55cf6679550c5e10cf8e45dace
***************
*** 13,27 ****
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Facturas pagas por parceiro: Operação inválida: O parceiro 'nãoexiste' não existe.
! Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Abrir
  2 - Guardar
--- 13,28 ----
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Número inválido!
! Escolha uma opção: Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu Principal
  1 - Abrir
  2 - Guardar
