*** testsFinal/A-18-02-M-ok.out	Wed Nov 10 23:23:51 2021
--- testsFinal/A-18-02-M-ok.outhyp	Thu Nov 11 11:58:04 2021
***************
*** 27,33 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
  3 - Registar venda
--- 27,51 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Gestão de Transacções
! 1 - Visualizar
! 2 - Registar desagregação
! 3 - Registar venda
! 4 - Registar compra
! 5 - Receber pagamento de venda
! 0 - Sair
! Escolha uma opção: Número inválido!
! Escolha uma opção: Opção inválida!
! Gestão de Transacções
! 1 - Visualizar
! 2 - Registar desagregação
! 3 - Registar venda
! 4 - Registar compra
! 5 - Receber pagamento de venda
! 0 - Sair
! Escolha uma opção: Número inválido!
! Escolha uma opção: Opção inválida!
! Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
  3 - Registar venda
***************
*** 71,78 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: VENDA|0|S1|SAL|10|1000|1000|10
! Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
  3 - Registar parceiro
--- 89,95 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
  3 - Registar parceiro
***************
*** 80,86 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
  3 - Registar parceiro
--- 97,104 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Número inválido!
! Escolha uma opção: Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
  3 - Registar parceiro
***************
*** 88,93 ****
--- 106,112 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu Principal
  1 - Abrir
  2 - Guardar
