*** tests/A-01-09-M-ok.out	2021-10-10 19:32:39.000000000 +0100
--- tests/A-01-09-M-ok.outhyp	2021-10-28 16:38:35.909875837 +0100
***************
*** 20,26 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Data actual: 20
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 20,26 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Data actual: 0
  Menu Principal
  1 - Abrir
  2 - Guardar
