*** tests/A-01-02-M-ok.out	2021-10-10 16:52:11.000000000 +0100
--- tests/A-01-02-M-ok.outhyp	2021-10-28 17:52:18.883599215 +0100
***************
*** 39,45 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu Principal
  1 - Abrir
  2 - Guardar
  3 - Mostrar Data
--- 39,45 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Guardar ficheiro como: Menu Principal
  1 - Abrir
  2 - Guardar
  3 - Mostrar Data
