*** tests/A-01-09-M-ok.out	2021-11-11 19:22:31.466059575 +0000
--- tests/A-01-09-M-ok.outhyp	2021-11-11 20:19:19.710202749 +0000
***************
*** 20,26 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Data actual: 20
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 20,26 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Data actual: 0
  Menu Principal
  1 - Abrir
  2 - Guardar
