*** tests/A-01-11-M-ok.out	2021-11-06 09:35:15.599020135 +0000
--- tests/A-01-11-M-ok.outhyp	2021-11-14 23:31:17.956145549 +0000
***************
*** 47,52 ****
--- 47,56 ----
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  NEW|ROLHA|10
+ NEW|VIDRO|1
+ NEW|SAL|100
+ NEW|GARRAFA|10
+ NEW|ROLHA|10
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
***************
*** 57,62 ****
--- 61,70 ----
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  NEW|ROLHA|10
+ NEW|VIDRO|1
+ NEW|SAL|100
+ NEW|GARRAFA|10
+ NEW|ROLHA|10
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
