<<<<<<< HEAD
*** tests/A-03-02-M-ok.out	Wed Oct 27 18:55:42 2021
--- tests/A-03-02-M-ok.outhyp	Thu Oct 28 16:59:08 2021
=======
*** tests/A-03-02-M-ok.out	2021-10-15 13:02:20.000000000 +0100
--- tests/A-03-02-M-ok.outhyp	2021-10-28 16:38:37.114477837 +0100
>>>>>>> 527c7f96c66f34d1382290d86ced60cd105a9d36
***************
*** 15,22 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: HIDROGENIO|200|5000
! Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
  3 - Visualizar lotes fornecidos por parceiro
--- 15,21 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
  3 - Visualizar lotes fornecidos por parceiro
