*** tests/A-10-11-M-ok.out	2021-11-05 13:01:33.224701157 +0000
--- tests/A-10-11-M-ok.outhyp	2021-11-14 23:31:22.556145738 +0000
***************
*** 106,111 ****
--- 106,115 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
+ NEW|ROLHA|2000
+ NEW|HIDROGENIO|200
+ NEW|VIDRO|1000
+ NEW|SAL|30
  NEW|SAL|30
  Gestão de Parceiros
  1 - Mostrar parceiro
***************
*** 116,121 ****
--- 120,129 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
+ NEW|ROLHA|2000
+ NEW|HIDROGENIO|200
+ NEW|VIDRO|1000
+ NEW|SAL|30
  NEW|SAL|30
  Gestão de Parceiros
  1 - Mostrar parceiro
