*** tests/A-10-19-M-ok.out	2021-11-05 13:01:35.774383211 +0000
--- tests/A-10-19-M-ok.outhyp	2021-11-14 23:31:23.812145790 +0000
***************
*** 51,56 ****
--- 51,60 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|500|0|0
+ NEW|ROLHA|2000
+ NEW|HIDROGENIO|200
+ NEW|VIDRO|1000
+ NEW|SAL|50
  BARGAIN|SAL|50
  Gestão de Parceiros
  1 - Mostrar parceiro
***************
*** 61,66 ****
--- 65,74 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
+ NEW|ROLHA|2000
+ NEW|HIDROGENIO|200
+ NEW|VIDRO|1000
+ NEW|SAL|50
  BARGAIN|SAL|50
  Gestão de Parceiros
  1 - Mostrar parceiro
***************
*** 144,150 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! BARGAIN|SAL|50
  BARGAIN|SAL|10
  Gestão de Parceiros
  1 - Mostrar parceiro
--- 152,162 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! NEW|ROLHA|2000
! NEW|HIDROGENIO|200
! NEW|VIDRO|1000
! NEW|SAL|10
! BARGAIN|SAL|10
  BARGAIN|SAL|10
  Gestão de Parceiros
  1 - Mostrar parceiro
