*** tests/A-10-06-M-ok.out	2021-11-15 03:04:18.545092306 +0000
--- tests/A-10-06-M-ok.outhyp	2021-11-15 03:06:21.150369383 +0000
***************
*** 50,56 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! BARGAIN|SAL|75
  BARGAIN|SAL|50
  Gestão de Parceiros
  1 - Mostrar parceiro
--- 50,56 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! BARGAIN|SAL|50
  BARGAIN|SAL|50
  Gestão de Parceiros
  1 - Mostrar parceiro
***************
*** 61,67 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! BARGAIN|SAL|75
  BARGAIN|SAL|50
  Gestão de Parceiros
  1 - Mostrar parceiro
--- 61,67 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! BARGAIN|SAL|50
  BARGAIN|SAL|50
  Gestão de Parceiros
  1 - Mostrar parceiro
