*** tests/A-10-20-M-ok.out	2021-11-05 13:01:36.152472805 +0000
--- tests/A-10-20-M-ok.outhyp	2021-11-14 23:31:23.984145797 +0000
***************
*** 180,187 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! NEW|SAL|10
! BARGAIN|ROLHA|100
  NEW|SAL|20
  BARGAIN|ROLHA|50
  Gestão de Parceiros
--- 180,191 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! NEW|ROLHA|50
! NEW|HIDROGENIO|200
! NEW|VIDRO|1000
! NEW|SAL|20
! NEW|SAL|20
! BARGAIN|ROLHA|50
  NEW|SAL|20
  BARGAIN|ROLHA|50
  Gestão de Parceiros
***************
*** 193,200 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! NEW|SAL|10
! BARGAIN|ROLHA|100
  NEW|SAL|20
  BARGAIN|ROLHA|50
  Gestão de Parceiros
--- 197,208 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! NEW|ROLHA|50
! NEW|HIDROGENIO|200
! NEW|VIDRO|1000
! NEW|SAL|20
! NEW|SAL|20
! BARGAIN|ROLHA|50
  NEW|SAL|20
  BARGAIN|ROLHA|50
  Gestão de Parceiros
