*** tests/A-10-17-M-ok.out	2021-11-05 13:01:35.152107712 +0000
--- tests/A-10-17-M-ok.outhyp	2021-11-14 23:31:23.532145778 +0000
***************
*** 81,86 ****
--- 81,90 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
+ NEW|ROLHA|2000
+ NEW|HIDROGENIO|200
+ NEW|VIDRO|1000
+ NEW|SAL|5
  NEW|SAL|5
  Gestão de Parceiros
  1 - Mostrar parceiro
***************
*** 91,96 ****
--- 95,104 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
+ NEW|ROLHA|2000
+ NEW|HIDROGENIO|200
+ NEW|VIDRO|1000
+ NEW|SAL|5
  NEW|SAL|5
  Gestão de Parceiros
  1 - Mostrar parceiro
