*** tests/A-10-18-M-ok.out	2021-11-05 13:01:35.460439345 +0000
--- tests/A-10-18-M-ok.outhyp	2021-11-14 23:11:23.272096403 +0000
***************
*** 113,120 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
- NEW|SAL|5
- NEW|SAL|30
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 113,118 ----
***************
*** 124,131 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
- NEW|SAL|5
- NEW|SAL|30
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 122,127 ----
***************
*** 135,141 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M3|M3|M3M3|NORMAL|0|0|0|0
- NEW|SAL|30
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 131,136 ----
