*** tests/A-10-12-M-ok.out	2021-11-05 13:01:33.522071408 +0000
--- tests/A-10-12-M-ok.outhyp	2021-11-14 23:11:22.300096363 +0000
***************
*** 54,60 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
- NEW|SAL|5
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 54,59 ----
***************
*** 64,70 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
- NEW|SAL|5
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 63,68 ----
