*** tests/A-11-02-M-ok.out	2021-10-21 16:21:02.502068640 +0100
--- tests/A-11-02-M-ok.outhyp	2021-11-14 23:31:24.300145810 +0000
***************
*** 82,87 ****
--- 82,91 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|2500|0|0
+ NEW|ROLHA|20
+ NEW|HIDROGENIO|200
+ NEW|VIDRO|1000
+ NEW|SAL|1000
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
