*** tests/A-11-06-M-ok.out	2021-11-15 03:04:18.549094306 +0000
--- tests/A-11-06-M-ok.outhyp	2021-11-15 03:06:24.207897509 +0000
***************
*** 75,80 ****
--- 75,81 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|200|0|0
+ NEW|papel|20
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
