*** tests/A-11-04-M-ok.out	2021-10-21 19:36:19.803330925 +0100
--- tests/A-11-04-M-ok.outhyp	2021-11-14 23:31:24.536145820 +0000
***************
*** 71,76 ****
--- 71,79 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|70000|0|0
+ NEW|ROLHA|20
+ NEW|VIDRO|30
+ NEW|GARRAFA|50
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
