*** tests/A-12-08-M-ok.out	2021-11-15 00:51:59.676344725 +0000
--- tests/A-12-08-M-ok.outhyp	2021-11-15 01:21:49.784418366 +0000
***************
*** 77,84 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 120
! Saldo contabilístico: 600
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 77,84 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 1020
! Saldo contabilístico: 9000
  Menu Principal
  1 - Abrir
  2 - Guardar
