*** tests/A-12-07-M-ok.out	2021-11-15 00:51:59.676344725 +0000
--- tests/A-12-07-M-ok.outhyp	2021-11-15 01:21:49.668418361 +0000
***************
*** 81,87 ****
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 20
! Saldo contabilístico: 238
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 81,87 ----
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 20
! Saldo contabilístico: 3300
  Menu Principal
  1 - Abrir
  2 - Guardar
***************
*** 118,125 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 53
! Saldo contabilístico: 238
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 118,125 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 50
! Saldo contabilístico: 3300
  Menu Principal
  1 - Abrir
  2 - Guardar
