*** tests/A-12-06-M-ok.out	2021-11-15 00:51:59.676344725 +0000
--- tests/A-12-06-M-ok.outhyp	2021-11-15 01:21:49.552418356 +0000
***************
*** 70,76 ****
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 130
! Saldo contabilístico: 200
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 70,76 ----
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 130
! Saldo contabilístico: 3000
  Menu Principal
  1 - Abrir
  2 - Guardar
***************
*** 108,114 ****
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 180
! Saldo contabilístico: 200
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 108,114 ----
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 180
! Saldo contabilístico: 3000
  Menu Principal
  1 - Abrir
  2 - Guardar
