*** tests/A-12-05-M-ok.out	2021-11-15 03:04:18.549094306 +0000
--- tests/A-12-05-M-ok.outhyp	2021-11-15 03:55:49.734042679 +0000
***************
*** 56,62 ****
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 0
! Saldo contabilístico: 200
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 56,62 ----
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 0
! Saldo contabilístico: 3000
  Menu Principal
  1 - Abrir
  2 - Guardar
