*** tests/A-12-06-M-ok.out	2021-11-12 10:10:35.546290073 +0000
--- tests/A-12-06-M-ok.outhyp	2021-11-14 23:11:25.332096488 +0000
***************
*** 70,76 ****
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 130
! Saldo contabilístico: 200
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 70,76 ----
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 130
! Saldo contabilístico: 3000
  Menu Principal
  1 - Abrir
  2 - Guardar
***************
*** 108,114 ****
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 180
! Saldo contabilístico: 200
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 108,114 ----
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 180
! Saldo contabilístico: 3000
  Menu Principal
  1 - Abrir
  2 - Guardar
