*** tests/A-12-05-M-ok.out	2021-11-15 00:51:59.676344725 +0000
--- tests/A-12-05-M-ok.outhyp	2021-11-15 01:21:49.420418351 +0000
***************
*** 56,62 ****
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 0
! Saldo contabilístico: 200
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 56,62 ----
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 0
! Saldo contabilístico: 3000
  Menu Principal
  1 - Abrir
  2 - Guardar
