*** tests/A-12-07-M-ok.out	2021-11-15 03:04:18.549094306 +0000
--- tests/A-12-07-M-ok.outhyp	2021-11-18 16:42:31.664568018 +0000
***************
*** 81,87 ****
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 20
! Saldo contabilístico: 238
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 81,87 ----
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 20
! Saldo contabilístico: 230
  Menu Principal
  1 - Abrir
  2 - Guardar
***************
*** 119,125 ****
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 53
! Saldo contabilístico: 238
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 119,125 ----
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 53
! Saldo contabilístico: 230
  Menu Principal
  1 - Abrir
  2 - Guardar
