*** tests/A-12-09-M-ok.out	2021-11-15 03:04:18.549094306 +0000
--- tests/A-12-09-M-ok.outhyp	2021-11-15 03:55:50.182266697 +0000
***************
*** 78,84 ****
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 20
! Saldo contabilístico: 140
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 78,84 ----
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 20
! Saldo contabilístico: 2000
  Menu Principal
  1 - Abrir
  2 - Guardar
