*** tests/A-14-02-M-ok.out	Fri Nov 12 09:03:28 2021
--- tests/A-14-02-M-ok.outhyp	Fri Nov 12 12:28:57 2021
***************
*** 71,80 ****
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Abrir
  2 - Guardar
--- 71,81 ----
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu Principal
  1 - Abrir
  2 - Guardar
