*** tests/A-15-07-M-ok.out	2021-10-31 22:37:40.682736993 +0000
--- tests/A-15-07-M-ok.outhyp	2021-11-14 23:59:46.988215854 +0000
***************
*** 115,121 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|200|2|ROLHA:1#SAL:5#VIDRO:10
  HIDROGENIO|200|5000
  ROLHA|2|3
  SAL|16|15
--- 115,121 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|200|5|ROLHA:1#SAL:5#VIDRO:10
  HIDROGENIO|200|5000
  ROLHA|2|3
  SAL|16|15
