*** tests/A-15-10-M-ok.out	2021-10-31 23:57:26.927950104 +0000
--- tests/A-15-10-M-ok.outhyp	2021-11-14 23:11:28.108096602 +0000
***************
*** 23,29 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: DESAGREGAÇÃO|0|R1|GARRAFA|10|-820|0|0|ROLHA:10:20#SAL:50:800#VIDRO:100:1000
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 23,29 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: DESAGREGAÇÃO|0|R1|GARRAFA|10|0|0|0|ROLHA:10:20#SAL:50:800#VIDRO:100:1000
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 50,61 ****
  0 - Sair
  Escolha uma opção: GARRAFA|S1|100|5
  HIDROGENIO|S1|200|5000
! ROLHA|R1|2|10
  ROLHA|R1|2|500
! SAL|R1|16|50
  SAL|S1|16|500
  VIDRO|M1|10|500
- VIDRO|R1|10|100
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
--- 50,61 ----
  0 - Sair
  Escolha uma opção: GARRAFA|S1|100|5
  HIDROGENIO|S1|200|5000
! GARRAFA|R1|2|10
  ROLHA|R1|2|500
! GARRAFA|R1|16|50
  SAL|S1|16|500
+ GARRAFA|R1|10|100
  VIDRO|M1|10|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
***************
*** 64,82 ****
  0 - Sair
  Escolha uma opção: GARRAFA|S1|100|5
  HIDROGENIO|S1|200|5000
! ROLHA|R1|2|10
  ROLHA|R1|2|500
! SAL|R1|16|50
  SAL|S1|16|500
  VIDRO|M1|10|500
- VIDRO|R1|10|100
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|100|5|ROLHA:1#SAL:5#VIDRO:10
  HIDROGENIO|200|5000
  ROLHA|2|510
  SAL|16|550
--- 64,82 ----
  0 - Sair
  Escolha uma opção: GARRAFA|S1|100|5
  HIDROGENIO|S1|200|5000
! GARRAFA|R1|2|10
  ROLHA|R1|2|500
! GARRAFA|R1|16|50
  SAL|S1|16|500
+ GARRAFA|R1|10|100
  VIDRO|M1|10|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|100|15|ROLHA:1#SAL:5#VIDRO:10
  HIDROGENIO|200|5000
  ROLHA|2|510
  SAL|16|550
