*** tests/A-16-03-M-ok.out	2021-11-15 03:04:18.553096305 +0000
--- tests/A-16-03-M-ok.outhyp	2021-11-15 03:55:53.267808825 +0000
***************
*** 71,77 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|S1|SAL|6|30|30|10|9
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 71,77 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|S1|SAL|6|30|27|10|9
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 86,92 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|S1|SAL|6|30|30|10|9
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 86,92 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|S1|SAL|6|30|27|10|9
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 105,112 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 30
! Saldo contabilístico: 30
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 105,112 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 27
! Saldo contabilístico: 180
  Menu Principal
  1 - Abrir
  2 - Guardar
***************
*** 126,132 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|300|0|30|30
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 126,132 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|300|0|30|27
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
***************
*** 137,143 ****
  0 - Sair
  Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|0|0|0
  R1|António Figueiredo|Lisboa|NORMAL|0|0|0|0
! S1|Toshiba|Tokyo, Japan|NORMAL|300|0|30|30
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 137,143 ----
  0 - Sair
  Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|0|0|0
  R1|António Figueiredo|Lisboa|NORMAL|0|0|0|0
! S1|Toshiba|Tokyo, Japan|NORMAL|300|0|30|27
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
