*** tests/A-16-05-M-ok.out	2021-11-15 03:04:18.553096305 +0000
--- tests/A-16-05-M-ok.outhyp	2021-11-15 03:55:53.491920833 +0000
***************
*** 97,103 ****
  0 - Sair
  Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|0|0|0
  R1|António Figueiredo|Lisboa|NORMAL|0|0|0|0
! S1|Toshiba|Tokyo, Japan|NORMAL|120|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 97,103 ----
  0 - Sair
  Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|0|0|0
  R1|António Figueiredo|Lisboa|NORMAL|0|0|0|0
! S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
