*** tests/A-18-03-M-ok.out	Fri Nov 12 09:03:28 2021
--- tests/A-18-03-M-ok.outhyp	Fri Nov 12 11:52:12 2021
***************
*** 50,56 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: DESAGREGAÇÃO|0|M2|GARRAFA|5|30|30|0|ROLHA:5:10#VIDRO:10:10
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 50,56 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: DESAGREGAÇÃO|0|M2|GARRAFA|5|30|30|0|ROLHA:5:10.0#VIDRO:10:10.0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
