*** tests/A-18-03-M-ok.out	2021-11-12 12:29:46.373589477 +0000
--- tests/A-18-03-M-ok.outhyp	2021-11-12 18:25:23.424970218 +0000
***************
*** 50,56 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: DESAGREGAÇÃO|0|M2|GARRAFA|5|30|30|0|ROLHA:5:10#VIDRO:10:10
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 50,56 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: DESAGREGAÇÃO|0|M2|GARRAFA|5|50|50|0|ROLHA:5:0#VIDRO:10:0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
