*** tests/A-18-02-M-ok.out	2021-11-12 12:29:46.373589477 +0000
--- tests/A-18-02-M-ok.outhyp	2021-11-12 16:14:52.609994823 +0000
***************
*** 71,78 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: VENDA|0|S1|SAL|10|1000|1000|10
! Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
  3 - Registar parceiro
--- 71,77 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
  3 - Registar parceiro
