*** tests/A-19-06-M-ok.out	2021-11-12 12:29:46.373589477 +0000
--- tests/A-19-06-M-ok.outhyp	2021-11-12 19:00:38.213836517 +0000
***************
*** 23,29 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|M1|GARRAFA|4|81|81|2
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 23,29 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|M1|GARRAFA|4|188|188|2
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 48,57 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|28|0|ROLHA:10#VIDRO:5
! ROLHA|2|55
  SAL|40|500
! VIDRO|1|35
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
--- 48,57 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|13|2|ROLHA:10#VIDRO:5
! ROLHA|2|75
  SAL|40|500
! VIDRO|1|45
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
