*** tests/A-19-02-M-ok.out	Thu Nov 11 19:38:17 2021
--- tests/A-19-02-M-ok.outhyp	Thu Nov 11 22:38:27 2021
***************
*** 16,29 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
  3 - Registar venda
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Data limite de pagamento: Identificador do produto: Quantidade: Registar venda: Operação inválida: Produto 'VIDRO': pedido=40, existências=20
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 16,30 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
  3 - Registar venda
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Número inválido!
! Escolha uma opção: Identificador do parceiro: Identificador do produto: Quantidade: Registar desagregação: Operação inválida: O parceiro 'VIDRO' não existe.
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 31,45 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|M1|VIDRO|30|30|30|2
! Gestão de Transacções
! 1 - Visualizar
! 2 - Registar desagregação
! 3 - Registar venda
! 4 - Registar compra
! 5 - Receber pagamento de venda
! 0 - Sair
! Escolha uma opção: Identificador da transacção: Visualizar: Operação inválida: A transacção '1' não existe.
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 32,39 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Número inválido!
! Escolha uma opção: Identificador do parceiro: Identificador do produto: Quantidade: Registar desagregação: Operação inválida: O parceiro 'VIDRO' não existe.
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 58,61 ****
--- 52,67 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
+ Escolha uma opção: Ficheiro a abrir: Abrir: Operação inválida: Problema ao abrir '1'.
+ Menu Principal
+ 1 - Abrir
+ 2 - Guardar
+ 3 - Mostrar Data
+ 4 - Avançar Data
+ 5 - Gestão de Produtos
+ 6 - Gestão de Parceiros
+ 7 - Gestão de Transacções
+ 8 - Menu de Consultas
+ 9 - Ver Saldo Global
+ 0 - Sair
  Escolha uma opção: 
\ No newline at end of file
