*** tests/A-20-11-M-ok.out	2021-11-15 03:04:18.553096305 +0000
--- tests/A-20-11-M-ok.outhyp	2021-11-18 03:03:29.878526004 +0000
***************
*** 178,184 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|2|M1|SAL|1|100|160|2|8
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 178,184 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|2|M1|SAL|1|100|190|2|8
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
