*** tests/A-20-01-M-ok.out	2021-11-15 03:04:18.553096305 +0000
--- tests/A-20-01-M-ok.outhyp	2021-11-15 03:55:56.757552968 +0000
***************
*** 63,69 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: M1|Rohit Figueiredo|New Delhi, India|NORMAL|900|0|100|90
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 63,69 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: M1|Rohit Figueiredo|New Delhi, India|NORMAL|1000|0|100|90
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
