*** tests/A-20-11-M-ok.out	2021-11-15 03:04:18.553096305 +0000
--- tests/A-20-11-M-ok.outhyp	2021-11-18 16:38:18.184557591 +0000
***************
*** 213,220 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|0|200|260
! M2|Toshiba|Tokyo, Japan|NORMAL|0|0|200|215
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
--- 213,220 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|1000|0|200|260
! M2|Toshiba|Tokyo, Japan|NORMAL|1000|0|200|215
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
