*** tests/A-20-12-M-ok.out	2021-11-15 03:04:18.553096305 +0000
--- tests/A-20-12-M-ok.outhyp	2021-11-15 03:55:58.174261026 +0000
***************
*** 133,139 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|2|M1|GARRAFA|50|500|475|2
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 133,139 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|2|M1|GARRAFA|50|500|500|2
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 141,147 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|3|M2|SAL|10|1000|900|7
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 141,147 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|3|M2|SAL|10|1000|1000|7
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 179,185 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|2|M1|GARRAFA|50|500|475|2|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 179,185 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|2|M1|GARRAFA|50|500|500|2|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 206,213 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|SELECTION|6850|0|710|685
! M2|Toshiba|Tokyo, Japan|SELECTION|11100|0|1210|1110
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
--- 206,213 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|7100|0|710|710
! M2|Toshiba|Tokyo, Japan|NORMAL|12100|0|1210|1110
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
